### PR TITLE
[bulk] validation: fix validation issues JM 5/9

### PR DESCRIPTION
**Pull request by jmart1428**
This PR fixes 109 issues.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/AspNetDocs?pullRequest=647) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/AspNetDocs?pullRequest=647) before we can accept your contribution.<br/><br/>:x: jmart1428 [sign now](https://cla.dotnetfoundation.org/dotnet/AspNetDocs?pullRequest=647)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/AspNetDocs?pullRequest=647) it.</sub>

File: aspnet/signalr/overview/performance/scaleout-with-redis.md [hunk 1]
```diff
@@ -74,11 +74,11 @@ Download the Ubuntu ISO from [http://www.ubuntu.com](http://www.ubuntu.com/).
 
 In Hyper-V, add a new VM. In the **Connect Virtual Hard Disk** step, select **Create a virtual hard disk**.
 
-![](scaleout-with-redis/_static/image2.png)
+![Window titled New Virtual Machine Wizard. 'Connect Virtual Hard Disk' is highlighted and 'Create a virtual hard disk' is selected.](scaleout-with-redis/_static/image2.png)
```
Comment (RedZephyr13) [inline aspnet/signalr/overview/performance/scaleout-with-redis.md:77]: Please leave out quotations.

File: aspnet/signalr/overview/performance/scaleout-with-redis.md [hunk 2]
```diff
@@ -74,11 +74,11 @@ Download the Ubuntu ISO from [http://www.ubuntu.com](http://www.ubuntu.com/).
...
 
 In the **Installation Options** step, select **Image file (.iso)**, click **Browse**, and browse to the Ubuntu installation ISO.
 
-![](scaleout-with-redis/_static/image3.png)
+![Window titled New Virtual Machine Wizard. 'Installation Options' is highlighted. 'Install...from a boot CD/DVD-ROM' and 'Image file (.iso):' are selected"](scaleout-with-redis/_static/image3.png)
```
Comment (RedZephyr13) [inline aspnet/signalr/overview/performance/scaleout-with-redis.md:81]: Let's get rid of the parenthesis and quotations and multiple periods.

File: aspnet/signalr/overview/performance/scaleout-with-redis.md [hunk 3]
```diff
@@ -130,15 +130,15 @@ Prepare your Windows Server instances to deploy the SignalR application.
 
 Add the IIS role. Include "Application Development" features, including the WebSocket Protocol.
 
-![](scaleout-with-redis/_static/image5.png)
+![Window titled 'Add Roles and Features Wizard'. On the 'Server Roles' option, 'WebSocket Protocol' is selected.](scaleout-with-redis/_static/image5.png)
```
Comment (RedZephyr13) [inline aspnet/signalr/overview/performance/scaleout-with-redis.md:133]: Let's remove the quotes.

File: aspnet/signalr/overview/performance/scaleout-with-redis.md [hunk 4]
```diff
@@ -130,15 +130,15 @@ Prepare your Windows Server instances to deploy the SignalR application.
...
 
 Also include the Management Service (listed under "Management Tools").
 
-![](scaleout-with-redis/_static/image6.png)
+![Window titled 'Add Roles and Features Wizard'. On the 'Server Roles' option, 'Management Service' is selected.](scaleout-with-redis/_static/image6.png)
```
Comment (RedZephyr13) [inline aspnet/signalr/overview/performance/scaleout-with-redis.md:137]: Let's remove the quotes.

File: aspnet/signalr/overview/performance/scaleout-with-sql-server.md [hunk 5]
```diff
@@ -72,7 +72,7 @@ To check whether Service Broker is enabled, query the **is\_broker\_enabled** co
 
 [!code-sql[Main](scaleout-with-sql-server/samples/sample2.sql)]
 
-![](scaleout-with-sql-server/_static/image3.png)
+![Window displaying the sys.databases catalog view.](scaleout-with-sql-server/_static/image3.png)
```
Comment (RedZephyr13) [inline aspnet/signalr/overview/performance/scaleout-with-sql-server.md:75]: For sys.databases, maybe use sys dot databases.

File: aspnet/signalr/overview/performance/scaleout-with-windows-azure-service-bus.md [hunk 6]
```diff
@@ -68,17 +68,17 @@ Start Visual Studio. From the **File** menu, click **New Project**.
...
 
 Hover the mouse over the new role, so the pencil icon visible. Click this icon to rename the role. Name the role "SignalRChat" and click **OK**.
 
-![](scaleout-with-windows-azure-service-bus/_static/image5.png)
+![Window titled 'New Windows Azure Cloud Service'. ASP.NET Web Role in the box on the right is labeled 'SignalRChat'.](scaleout-with-windows-azure-service-bus/_static/image5.png)
 
 In the **New ASP.NET Project** dialog, select **MVC**, and click OK.
 
-![](scaleout-with-windows-azure-service-bus/_static/image6.png)
+![Window titled 'New ASP.NET Project - SignalRChat'. Under 'Select a template', 'MVC' is selected.](scaleout-with-windows-azure-service-bus/_static/image6.png)
```
Comment (RedZephyr13) [inline aspnet/signalr/overview/performance/scaleout-with-windows-azure-service-bus.md:81]: let's get rid of the dash, or spell it out.  also change ASP.NET to A S P dot NET.  So also change MVC to M V C.

File: aspnet/signalr/overview/performance/scaleout-with-windows-azure-service-bus.md [hunk 7]
```diff
@@ -103,33 +103,33 @@ In your application's Startup.cs file, add the following code:
...
+![Window titled 'Connect to your namespace' with options underneath.](scaleout-with-windows-azure-service-bus/_static/image8.png)
 
 [!code-csharp[Main](scaleout-with-windows-azure-service-bus/samples/sample4.cs)]
 
 ## Deploy to Azure
 
 In Solution Explorer, expand the **Roles** folder inside the ChatService project.
 
-![](scaleout-with-windows-azure-service-bus/_static/image9.png)
+![The folder 'Roles' is open and 'SignalRChat' is selected.](scaleout-with-windows-azure-service-bus/_static/image9.png)
```
Comment (RedZephyr13) [inline aspnet/signalr/overview/performance/scaleout-with-windows-azure-service-bus.md:118]: Let's change SignalRChat to Signal R Chat.

File: aspnet/signalr/overview/performance/scaleout-with-windows-azure-service-bus.md [hunk 8]
```diff
@@ -103,33 +103,33 @@ In your application's Startup.cs file, add the following code:
...
 
 Right-click the SignalRChat role and select **Properties**. Select the **Configuration** tab. Under **Instances** select 2. You can also set the VM size to **Extra Small**.
 
-![](scaleout-with-windows-azure-service-bus/_static/image10.png)
+![Under Instances, the Instance count is set to 2 and VM Size is set to Extra small.](scaleout-with-windows-azure-service-bus/_static/image10.png)
```
Comment (RedZephyr13) [inline aspnet/signalr/overview/performance/scaleout-with-windows-azure-service-bus.md:122]: Change VM to V M.

File: aspnet/signalr/overview/performance/scaleout-with-redis.md [hunk 9]
```diff
@@ -36,7 +36,7 @@ In this tutorial, you will use [Redis](http://redis.io/) to distribute messages
 
 Redis is an in-memory key-value store. It also supports a messaging system with a publish/subscribe model. The SignalR Redis backplane uses the pub/sub feature to forward messages to other servers.
 
-![Image showing arrows going from Redis Server to VMs and then to computers. One arrow labeled "Publish" goes from VMs to Redis Server.](scaleout-with-redis/_static/image1.png)
+![Diagram that shows arrows going from Redis Server to V M and then to computers One arrow labeled Publish goes from V M to Redis Server.](scaleout-with-redis/_static/image1.png)
```
Comment (RedZephyr13) [inline aspnet/signalr/overview/performance/scaleout-with-redis.md:39]: Let's split up this sentence.
"Diagram that shows arrows going from Redis Server to V M and then to computers. One arrow labeled Publish goes from V M to Redis Server."

File: aspnet/signalr/overview/performance/scaleout-with-redis.md [hunk 10]
```diff
@@ -74,11 +74,11 @@ Download the Ubuntu ISO from [http://www.ubuntu.com](http://www.ubuntu.com/).
 
 In Hyper-V, add a new VM. In the **Connect Virtual Hard Disk** step, select **Create a virtual hard disk**.
 
-![Window titled New Virtual Machine Wizard. 'Connect Virtual Hard Disk' is highlighted and 'Create a virtual hard disk' is selected.](scaleout-with-redis/_static/image2.png)
+![Screenshot of a window titled New Virtual Machine Wizard and Connect Virtual Hard Disk and Create a virtual hard disk are selected.](scaleout-with-redis/_static/image2.png)
```
Comment (RedZephyr13) [inline aspnet/signalr/overview/performance/scaleout-with-redis.md:77]: Let's split this sentence up to.  "Screenshot that shows the New Virtual Machine Wizard. Connect Virtual Hard Disk and Create a virtual hard disk are selected.

File: aspnet/signalr/overview/performance/scaleout-with-redis.md [hunk 11]
```diff
@@ -74,11 +74,11 @@ Download the Ubuntu ISO from [http://www.ubuntu.com](http://www.ubuntu.com/).
...
 
 In the **Installation Options** step, select **Image file (.iso)**, click **Browse**, and browse to the Ubuntu installation ISO.
 
-![Window titled New Virtual Machine Wizard. 'Installation Options' is highlighted. 'Install...from a boot CD/DVD-ROM' and 'Image file (.iso):' are selected"](scaleout-with-redis/_static/image3.png)
+![Screenshot that shows a window titled New Virtual Machine Wizard Installation Options is highlighted Install from a boot C D D V D ROM and Image file dot i s o are selected](scaleout-with-redis/_static/image3.png)
```
Comment (RedZephyr13) [inline aspnet/signalr/overview/performance/scaleout-with-redis.md:81]: How about, "Screenshot that shows New Virtual Machine Wizard on the Installation Options page. Install an O S from a boot C D D V D ROM and Image file dot iso are selected."

File: aspnet/signalr/overview/performance/scaleout-with-redis.md [hunk 12]
```diff
@@ -96,7 +96,7 @@ Now start the Redis server:
 
 [!code-css[Main](scaleout-with-redis/samples/sample4.css)]
 
-![Interface displays main page of Redis server.](scaleout-with-redis/_static/image4.png)
+![Screenshot that shows main page of Redis server.](scaleout-with-redis/_static/image4.png)
```
Comment (RedZephyr13) [inline aspnet/signalr/overview/performance/scaleout-with-redis.md:99]: Maybe, "Screenshot that shows the main page of the Redis server."

File: aspnet/signalr/overview/performance/scaleout-with-redis.md [hunk 13]
```diff
@@ -130,15 +130,15 @@ Prepare your Windows Server instances to deploy the SignalR application.
 
 Add the IIS role. Include "Application Development" features, including the WebSocket Protocol.
 
-![Window titled 'Add Roles and Features Wizard'. On the 'Server Roles' option, 'WebSocket Protocol' is selected.](scaleout-with-redis/_static/image5.png)
+![Screenshot that shows a window titled Add Roles and Features Wizard On the Server Roles option WebSocket Protocol is selected.](scaleout-with-redis/_static/image5.png)
 
 Also include the Management Service (listed under "Management Tools").
 
-![Window titled 'Add Roles and Features Wizard'. On the 'Server Roles' option, 'Management Service' is selected.](scaleout-with-redis/_static/image6.png)
+![Screenshot that shows a window titled Add Roles and Features Wizard On the Server Roles option Management Service is selected.](scaleout-with-redis/_static/image6.png)
```
Comment (RedZephyr13) [inline aspnet/signalr/overview/performance/scaleout-with-redis.md:137]: How about, "Screenshot that shows Add Roles and Features Wizard. Management Service is selected."

File: aspnet/signalr/overview/performance/scaleout-with-redis.md [hunk 14]
```diff
@@ -130,15 +130,15 @@ Prepare your Windows Server instances to deploy the SignalR application.
...
 
 **Install Web Deploy 3.0.** When you run IIS Manager, it will prompt you to install Microsoft Web Platform, or you can [download the installer](https://go.microsoft.com/fwlink/?LinkId=255386). In the Platform Installer, search for Web Deploy and install Web Deploy 3.0
 
-![Window titled 'Web Platform Installer 4.5'. 'Web Deploy 3.0' is selected.](scaleout-with-redis/_static/image7.png)
+![Screenshot that shows a window titled Web Platform Installer 4 point 5 Web Deploy 3 point 0 is selected.](scaleout-with-redis/_static/image7.png)
```
Comment (RedZephyr13) [inline aspnet/signalr/overview/performance/scaleout-with-redis.md:141]: How about, "Screenshot that shows Web Platform Installer 4 point 5 dialog box. Web Deploy 3 point 0 is selected."

File: aspnet/signalr/overview/performance/scaleout-with-sql-server.md [hunk 15]
```diff
@@ -34,7 +34,7 @@ by [Mike Wasson](https://github.com/MikeWasson), [Patrick Fletcher](https://gith
 
 In this tutorial, you will use SQL Server to distribute messages across a SignalR application that is deployed in two separate IIS instances. You can also run this tutorial on a single test machine, but to get the full effect, you need to deploy the SignalR application to two or more servers. You must also install SQL Server on one of the servers, or on a separate dedicated server. Another option is to run the tutorial using VMs on Azure.
 
-![An image with arrows that go from SQL Server to VMs to computers. One arrow labeled "Update" starts at VMs and goes to SQL Server.](scaleout-with-sql-server/_static/image1.png)
+![Diagram that shows arrows going from S Q L Server to V M to computers One arrow labeled Update starts at V M and goes to S Q L Server.](scaleout-with-sql-server/_static/image1.png)
```
Comment (RedZephyr13) [inline aspnet/signalr/overview/performance/scaleout-with-sql-server.md:37]: Lets split up this sentence too. "Diagram that shows arrows going from S Q L Server to V M to computers. One arrow labeled Update starts at V M and goes to S Q L Server."

File: aspnet/signalr/overview/performance/scaleout-with-sql-server.md [hunk 16]
```diff
@@ -62,7 +62,7 @@ Decide whether the application will use Windows authentication or SQL Server aut
 
 Create a new database for the backplane to use. You can give the database any name. You don't need to create any tables in the database; the backplane will create the necessary tables.
 
-![Window titled 'Object Explorer'. The folder 'Databases' is selected.](scaleout-with-sql-server/_static/image2.png)
+![Screenshot of a window titled Object Explorer The folder labeled Databases is selected.](scaleout-with-sql-server/_static/image2.png)
```
Comment (RedZephyr13) [inline aspnet/signalr/overview/performance/scaleout-with-sql-server.md:65]: Let's split up this sentence too.

File: aspnet/signalr/overview/performance/scaleout-with-sql-server.md [hunk 17]
```diff
@@ -104,15 +104,15 @@ Prepare your Windows Server instances to deploy the SignalR application.
 
 Add the IIS role. Include "Application Development" features, including the WebSocket Protocol.
 
-![Window titled 'Add Roles and Features Wizard'. On the 'Server Roles' option, 'WebSocket Protocol' is selected.](scaleout-with-sql-server/_static/image4.png)
+![Window titled Add Roles and Features Wizard On the Server Roles option WebSocket Protocol is selected.](scaleout-with-sql-server/_static/image4.png)
```
Comment (RedZephyr13) [inline aspnet/signalr/overview/performance/scaleout-with-sql-server.md:107]: How about, "Screenshot that shows Add Roles and Features Wizard.  Web Socket Protocol is selected."

File: aspnet/signalr/overview/performance/scaleout-with-sql-server.md [hunk 18]
```diff
@@ -104,15 +104,15 @@ Prepare your Windows Server instances to deploy the SignalR application.
...
 
 Also include the Management Service (listed under "Management Tools").
 
-![](scaleout-with-sql-server/_static/image5.png)
+![Window titled Add Roles and Features Wizard On the Server Roles option Management Service is selected.](scaleout-with-sql-server/_static/image5.png)
```
Comment (RedZephyr13) [inline aspnet/signalr/overview/performance/scaleout-with-sql-server.md:111]: How about, "Screenshot that shows the Select server roles page.  Management Tools is selected.

File: aspnet/signalr/overview/performance/scaleout-with-windows-azure-service-bus.md [hunk 19]
```diff
@@ -17,7 +17,7 @@ by [Mike Wasson](https://github.com/MikeWasson), [Patrick Fletcher](https://gith
 
 In this tutorial, you will deploy a SignalR application to a Windows Azure Web Role, using the Service Bus backplane to distribute messages to each role instance. (You can also use the Service Bus backplane with [web apps in Azure App Service](/azure/app-service-web/).)
 
-![Image with arrows from 'Topic' to 'Web Roles' to computers. An arrow labeled "publish" starts at 'Web Roles' and goes to 'Topic'.](scaleout-with-windows-azure-service-bus/_static/image1.png)
+![Diagram that shows arrows from Topic to Web Roles to computers An arrow labeled publish starts at Web Roles and goes to Topic.](scaleout-with-windows-azure-service-bus/_static/image1.png)
```
Comment (RedZephyr13) [inline aspnet/signalr/overview/performance/scaleout-with-windows-azure-service-bus.md:20]: Let's split up this sentence too.

File: aspnet/signalr/overview/performance/scaleout-with-windows-azure-service-bus.md [hunk 20]
```diff
@@ -68,17 +68,17 @@ Start Visual Studio. From the **File** menu, click **New Project**.
 
 In the **New Project** dialog box, expand **Visual C#**. Under **Installed Templates**, select **Cloud** and then select **Windows Azure Cloud Service**. Keep the default .NET Framework 4.5. Name the application ChatService and click **OK**.
 
-![Window titled 'New Project'. 'Cloud' is selected and "ChatService" is typed under 'Name'.](scaleout-with-windows-azure-service-bus/_static/image4.png)
+![Screenshot of window with Cloud selected and ChatService typed under Name.](scaleout-with-windows-azure-service-bus/_static/image4.png)
```
Comment (RedZephyr13) [inline aspnet/signalr/overview/performance/scaleout-with-windows-azure-service-bus.md:71]: How about, "Screenshot that shows the New Project dialog box. Cloud is selected in the Office Share Point folder. Chat Service is in the Name field."

File: aspnet/signalr/overview/performance/scaleout-with-windows-azure-service-bus.md [hunk 21]
```diff
@@ -68,17 +68,17 @@ Start Visual Studio. From the **File** menu, click **New Project**.
...
 
 In the **New Windows Azure Cloud Service** dialog, select ASP.NET Web Role. Click the right-arrow button (**&gt;**) to add the role to your solution.
 
 Hover the mouse over the new role, so the pencil icon visible. Click this icon to rename the role. Name the role "SignalRChat" and click **OK**.
 
-![Window titled 'New Windows Azure Cloud Service'. ASP.NET Web Role in the box on the right is labeled 'SignalRChat'.](scaleout-with-windows-azure-service-bus/_static/image5.png)
+![Screenshot of window titled New Windows Azure Cloud Service and Signal R Chat is typed above A S P dot NET Web Role.](scaleout-with-windows-azure-service-bus/_static/image5.png)
```
Comment (RedZephyr13) [inline aspnet/signalr/overview/performance/scaleout-with-windows-azure-service-bus.md:77]: How about, "Screenshot that shows the New Windows Azure Cloud Service dialog box. Signal R Chat is typed above A S P dot NET Web Role."

File: aspnet/signalr/overview/performance/scaleout-with-windows-azure-service-bus.md [hunk 22]
```diff
@@ -68,17 +68,17 @@ Start Visual Studio. From the **File** menu, click **New Project**.
...
 
 In the **New ASP.NET Project** dialog, select **MVC**, and click OK.
 
-![Window titled 'New ASP.NET Project - SignalRChat'. Under 'Select a template', 'MVC' is selected.](scaleout-with-windows-azure-service-bus/_static/image6.png)
+![Screenshot of window with M V C selected under Select a template.](scaleout-with-windows-azure-service-bus/_static/image6.png)
```
Comment (RedZephyr13) [inline aspnet/signalr/overview/performance/scaleout-with-windows-azure-service-bus.md:81]: How about, "Screenshot that shows the New A S P dot NET Project dialog box. MVC is the selected template."

File: aspnet/signalr/overview/performance/scaleout-with-windows-azure-service-bus.md [hunk 23]
```diff
@@ -103,51 +103,51 @@ In your application's Startup.cs file, add the following code:
 
 Now you need to get your service bus connection string. In the Azure portal, select the service bus namespace that you created and click the Access Key icon.
 
-![An image of a plus sign labeled 'Create', an image of a key labeled 'Access Key', and an image of a trash can labeled 'Delete'.](scaleout-with-windows-azure-service-bus/_static/image7.png)
+![Graphic of a plus sign labeled Create and a key labeled Access Key and a trash can labeled Delete.](scaleout-with-windows-azure-service-bus/_static/image7.png)
```
Comment (RedZephyr13) [inline aspnet/signalr/overview/performance/scaleout-with-windows-azure-service-bus.md:106]: This is a screenshot, so let's say, "Screenshot that shows a plus sign labeled Create, a key labeled Access Key, and a trash can labeled Delete."

File: aspnet/signalr/overview/performance/scaleout-with-windows-azure-service-bus.md [hunk 24]
```diff
@@ -103,51 +103,51 @@ In your application's Startup.cs file, add the following code:
...
 
 Copy the connection string to the clipboard, then paste it into the *connectionString* variable.
 
-![Window titled 'Connect to your namespace' with options underneath.](scaleout-with-windows-azure-service-bus/_static/image8.png)
+![Screenshot of Window titled Connect to your namespace with options underneath.](scaleout-with-windows-azure-service-bus/_static/image8.png)
```
Comment (RedZephyr13) [inline aspnet/signalr/overview/performance/scaleout-with-windows-azure-service-bus.md:110]: Let's change the beginning of your sentence to, "Screenshot of a window..."  Or, "Screenshot that shows the Access Key Connect to your namespace dialog box."

File: aspnet/signalr/overview/performance/scaleout-with-windows-azure-service-bus.md [hunk 25]
```diff
@@ -103,51 +103,51 @@ In your application's Startup.cs file, add the following code:
...
 
 [!code-csharp[Main](scaleout-with-windows-azure-service-bus/samples/sample4.cs)]
 
 ## Deploy to Azure
 
 In Solution Explorer, expand the **Roles** folder inside the ChatService project.
 
-![The folder 'Roles' is open and 'SignalRChat' is selected.](scaleout-with-windows-azure-service-bus/_static/image9.png)
+![Screenshot of an open folder titled Roles and Signal R Chat selected.](scaleout-with-windows-azure-service-bus/_static/image9.png)
```
Comment (RedZephyr13) [inline aspnet/signalr/overview/performance/scaleout-with-windows-azure-service-bus.md:118]: Let's add an "is" before selected.

File: aspnet/signalr/overview/performance/scaleout-with-windows-azure-service-bus.md [hunk 26]
```diff
@@ -103,51 +103,51 @@ In your application's Startup.cs file, add the following code:
...
 
-![Under Instances, the Instance count is set to 2 and VM Size is set to Extra small.](scaleout-with-windows-azure-service-bus/_static/image10.png)
+![Screenshot showing Instances and the Instance count is set to 2 and V M Size is set to Extra small.](scaleout-with-windows-azure-service-bus/_static/image10.png)
 
 Save the changes.
 
 In Solution Explorer, right-click the ChatService project. Select **Publish**.
 
-![A menu with 'Publish' selected.](scaleout-with-windows-azure-service-bus/_static/image11.png)
+![Screenshot of Chat Service menu with Publish selected.](scaleout-with-windows-azure-service-bus/_static/image11.png)
```
Comment (RedZephyr13) [inline aspnet/signalr/overview/performance/scaleout-with-windows-azure-service-bus.md:128]: How about, "Screenshot that shows Solution Explorer. Publish is selected in the Chat Service context menu."

File: aspnet/signalr/overview/performance/scaleout-with-windows-azure-service-bus.md [hunk 27]
```diff
@@ -103,51 +103,51 @@ In your application's Startup.cs file, add the following code:
...
 
 If this is your first time publishing to Windows Azure, you must download your credentials. In the **Publish** wizard, click "Sign in to download credentials". This will prompt you to sign into the Windows Azure portal and download a publish settings file.
 
-![Window titled 'Publish Windows Azure Application'. 'Sign in to download credentials' is circled in red.](scaleout-with-windows-azure-service-bus/_static/image12.png)
+![Screenshot of window with title windows azure publish sign in and Sign in to download credentials is circled in red.](scaleout-with-windows-azure-service-bus/_static/image12.png)
```
Comment (RedZephyr13) [inline aspnet/signalr/overview/performance/scaleout-with-windows-azure-service-bus.md:132]: How about, "Screenshot that shows the Publish Windows Azure Application dialog box.  Sign in to download credentials is circled in red."

File: aspnet/signalr/overview/performance/scaleout-with-windows-azure-service-bus.md [hunk 28]
```diff
@@ -103,51 +103,51 @@ In your application's Startup.cs file, add the following code:
...
 
 Click **Import** and select the publish settings file that you downloaded.
 
 Click **Next**. In the **Publish Settings** dialog, under **Cloud Service**, select the cloud service that you created earlier.
 
-![](scaleout-with-windows-azure-service-bus/_static/image13.png)
+![Screenshot of window with title Windows Azure Publish Settings and Settings is selected.](scaleout-with-windows-azure-service-bus/_static/image13.png)
```
Comment (RedZephyr13) [inline aspnet/signalr/overview/performance/scaleout-with-windows-azure-service-bus.md:138]: How about, "Screenshot that shows the Windows Azure Publish Settings page."

File: aspnet/signalr/overview/performance/scaleout-with-windows-azure-service-bus.md [hunk 29]
```diff
@@ -103,51 +103,51 @@ In your application's Startup.cs file, add the following code:
...
 
 The backplane automatically creates the topic and the subscriptions. To see the subscriptions and message activity, open the Azure portal, select the Service Bus namespace, and click on "Topics".
 
-![](scaleout-with-windows-azure-service-bus/_static/image14.png)
+![Screenshot of navigation menu with topics selected.](scaleout-with-windows-azure-service-bus/_static/image14.png)
 
 It make take a few minutes for the message activity to show up in the dashboard.
 
-![](scaleout-with-windows-azure-service-bus/_static/image15.png)
+![Screenshot that shows graph of subscription and message activity on a timeline.](scaleout-with-windows-azure-service-bus/_static/image15.png)
```
Comment (RedZephyr13) [inline aspnet/signalr/overview/performance/scaleout-with-windows-azure-service-bus.md:150]: Let's add an "a" before "graph."

File: aspnet/visual-studio/overview/2012/windows-azure-authentication.md [hunk 30]
```diff
@@ -95,37 +95,37 @@ To publish an application using Windows Azure Authentication to an Azure Web Sit
 
 1. Right click on your application and select **Publish:**
 
-    ![](windows-azure-authentication/_static/image3.jpg)
+    ![Screenshot that shows the WinAzureAuthDemo right click menu. Publish is selected.](windows-azure-authentication/_static/image3.jpg)
```
Comment (RedZephyr13) [inline aspnet/visual-studio/overview/2012/windows-azure-authentication.md:98]: Let's split up "WinAzureAuthDemo."

File: aspnet/single-page-application/overview/templates/emberjs-template.md [hunk 31]
```diff
@@ -21,7 +21,7 @@ The EmberJS SPA template is designed to get you started quickly building interac
 
 "Single-page application" (SPA) is the general term for a web application that loads a single HTML page and then updates the page dynamically, instead of loading new pages. After the initial page load, the SPA talks with the server through AJAX requests.
 
-![](emberjs-template/_static/image1.png)
+![Diagram that shows two boxes labeled Client and Server. An arrow labeled AJAX goes from Client to Server. An arrow labeled H T M L and an arrow labeled J S O N go from Server to Client.](emberjs-template/_static/image1.png)
```
Comment (RedZephyr13) [inline aspnet/single-page-application/overview/templates/emberjs-template.md:24]: I think JSON files are pronounced like "Jason," so we could use "J SON" instead here.

File: aspnet/single-page-application/overview/introduction/knockoutjs-template.md [hunk 32]
```diff
@@ -36,33 +36,33 @@ Start Visual Studio and select **New Project** from the Start page. Or, from the
...
 
 In the **New Project** wizard, select **Single Page Application**.
 
-![](knockoutjs-template/_static/image3.png)
+![Screenshot that shows the New A S P dot NET M V C 4 Project dialog box. The Single Page Application template is selected.](knockoutjs-template/_static/image3.png)
 
 Press F5 to build and run the application. When the application first runs, it displays a login screen.
 
-![](knockoutjs-template/_static/image4.png)
+![Screenshot that shows the My Todo List login screen.](knockoutjs-template/_static/image4.png)
```
Comment (RedZephyr13) [inline aspnet/single-page-application/overview/introduction/knockoutjs-template.md:47]: Let's split up "Todo" so it sounds like "To do."  Also on lines 55 and 120.

File: aspnet/signalr/overview/performance/scaleout-with-redis.md [hunk 33]
```diff
@@ -74,11 +74,11 @@ Download the Ubuntu ISO from [http://www.ubuntu.com](http://www.ubuntu.com/).
 
 In Hyper-V, add a new VM. In the **Connect Virtual Hard Disk** step, select **Create a virtual hard disk**.
 
-![Screenshot of a window titled New Virtual Machine Wizard and Connect Virtual Hard Disk and Create a virtual hard disk are selected.](scaleout-with-redis/_static/image2.png)
+![Screenshot that shows the New Virtual Machine Wizard dialog box. Connect Virtual Hard Disk and Create a virtual hard disk are selected.](scaleout-with-redis/_static/image2.png)
 
 In the **Installation Options** step, select **Image file (.iso)**, click **Browse**, and browse to the Ubuntu installation ISO.
 
-![Screenshot that shows a window titled New Virtual Machine Wizard Installation Options is highlighted Install from a boot C D D V D ROM and Image file dot i s o are selected](scaleout-with-redis/_static/image3.png)
+![Screenshot that shows the New Virtual Machine Wizard on the Installation Options page. Install an O S from a boot C D D V D ROM and Image file dot i s o are selected](scaleout-with-redis/_static/image3.png)
```
Comment (RedZephyr13) [inline aspnet/signalr/overview/performance/scaleout-with-redis.md:81]: I didn't catch this the first time, but let's add a period to the end of your sentence.

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**